shared/app.js

```diff
@@ -70,6 +73,8 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
+      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
```

Comment:
yeah, balls. sorry about that guys. is the requirejs example updated so i can use it as a test bed for this stuff?